website_form_builder/static/src/css/website_form_builder.less

```diff
@@ -22,3 +22,11 @@ body .modal.o_website_modal {
         }
     }
 }
+
+.modal.o_technical_modal {
```

Comment:
I think that you need to fix line 18 instead of adding these.

Reply:
I'd say is the right one

Reply:
@Yajo How do you do in line 18?

Reply:
AFAICS you just have to change `o_website_modal` for `o_technical_modal` in [line 18 of this same file](https://github.com/OCA/website/pull/646/files#diff-5e0c8f09edd6ac67691c8881ecc2c7d9R18).